USER-INTERFACE-INTERFACE.md

```diff
@@ -386,6 +405,66 @@ database password. If you want to know whether the password you have is the corr
   try to connect to when it starts up next time.  It's a secret, so if you don't supply the `dbPasswordOpt` in the
   request you won't see it.
 
+* `paymentCurves`: These are parameters that define the Node's regard to quality of financial records created during interaction with
+  other Nodes. The goal is to determine whether the records have crossed boarders meaning that a certain action needs to be taken 
+  - either to make a payment to the touched Nodes in order to redeem our debts or put those Nodes under ban since they've too long 
+  ignored their liabilities toward us. These are ever present values, no matter if the user's set any value, they have defaults.
+
+* `balanceDecreasesForSec`: This interval begins after payment_suggested_after_sec for payables and after 
```

Comment:
`This time interval -- in seconds -- begins...` would help to explain the `sec` abbreviation.